web/src/components/applications-page/application-list/application-list-item/index.tsx

```diff
@@ -140,56 +146,60 @@ export const ApplicationListItem: FC<ApplicationListItemProps> = memo(
           </TableCell>
           <TableCell>{APPLICATION_KIND_TEXT[app.kind]}</TableCell>
           <TableCell>
-            {app.labelsMap.length !== 0
-              ? app.labelsMap.map(([key, value]) => (
-                  <>
-                    <span>{key + ": " + value}</span>
-                    <br />
-                  </>
-                ))
-              : "-"}
+            <div className={classes.labels}>
+              {app.labelsMap.length !== 0
+                ? app.labelsMap.map(([key, value]) => (
+                    <>
+                      <span>{key + ": " + value}</span>
+                      <br />
+                    </>
+                  ))
+                : "-"}
+            </div>
           </TableCell>
           {recentlyDeployment ? (
             <>
-              <TableCell className={classes.version}>
-                {recentlyDeployment.versionsList.length !== 0 ? (
-                  recentlyDeployment.versionsList.map((v) =>
-                    v.name == "" ? (
-                      <>
-                        <span>{v.version}</span>
-                        <br />
-                      </>
-                    ) : (
-                      <>
-                        <Link
-                          href={v.url.includes("://") ? v.url : `//${v.url}`}
-                          target="_blank"
-                          rel="noreferrer"
-                        >
-                          {/* Trim first 7 characters like commit hash in case it is too long */}
-                          {v.name}:
-                          {v.version.length > 7
-                            ? `${v.version.slice(0, 7)}...`
-                            : v.version}
-                          <OpenInNewIcon className={classes.linkIcon} />
-                        </Link>
-                        <br />
-                      </>
+              <TableCell>
```

Comment:
I think `Artifact Versions` should be fixed as well as Labels for that support case.

Reply:
yes, it's. At L163 👀  We can not directly set max-height to TableCell so I use a trick to add div to the TableCell and set max-height there. Same thing I did at L149. (The demo video is only for labels column does not mean I did not update the versions column)

Reply:
Oops, I missed [this one](https://github.com/pipe-cd/pipecd/pull/3567/files#diff-561b45876d43b4ef62f2607495d990b27a408098d8dfbce07b28850cd79b3598R40).
Thanks.